sdk/storage/azblob/zt_examples_test.go

```diff
@@ -133,20 +114,21 @@ func Example() {
 		log.Fatal(err)
 	}
 
-	// Delete the blob we created earlier.
-	_, err = blockBlob.Delete(ctx, nil)
+	// Delete the blob.
+	_, err = blockBlob.Delete(context.TODO(), nil)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	// Delete the container we created earlier.
-	_, err = container.Delete(ctx, nil)
+	// Delete the container.
+	_, err = container.Delete(context.TODO(), nil)
 	if err != nil {
 		log.Fatal(err)
 	}
 }
 
 // Awaiting Mohit's test PR, StorageError is barren for now.
+// TODO: is this still valid?
```

Comment:
@mohsha-msft is this true? should this example be shown?

Reply:
That must be some old comment. Will remove it.

Reply:
For clarification, remove the comment or remove the example?

Reply:
Remove the comment. Not example.